core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -311,6 +311,9 @@ public enum Property {
       "The balancer class that accumulo will use to make tablet assignment and "
           + "migration decisions.",
       "1.3.5"),
+  MANAGER_TABLET_GROUP_WATCHER_INTERVAL("manager.tablet.watcher.interval", "60s",
+      PropertyType.TIMEDURATION,
+      "Time to wait between scanning tablet states to determine migrations, etc.", "2.1.2"),
```

Comment:
I'm not sure `etc.` is doing much to help the reader understand the category of things this affects. The migrations is only one example. If there were a second example in the list, then `etc.` could make sense, because one might be able to infer a category of tasks from two examples. If there's no second example to add to the list, you could just phrase it as "states to determine outstanding tasks, such as migrations."

```suggestion
      "Time to wait between scanning tablet states to identify outstanding tasks to perform, such as migrations.", "2.1.2"),
```

Reply:
Updated property description in ff6923e